rusk-wallet/src/lib/prompt.rs

```diff
@@ -480,12 +480,12 @@ fn request_gas_limit() -> u64 {
     let question = requestty::Question::int("amt")
         .message("Introduce the gas limit for this transaction:")
         .default(DEFAULT_GAS_LIMIT as i64)
-        .validate_on_key(|n, _| n > 0)
+        .validate_on_key(|n, _| n > (MIN_GAS_LIMIT as i64))
```

Comment:
The correct formula is `n > MIN_GAS_LIMIT + n * 0.07` (not taking into account the float implicit conversion)

Reply:
N/A